src/app/models/app-title.model.ts

```diff
@@ -0,0 +1,4 @@
+export interface AppTitleModel {
```

Comment:
Does this need a model?
Is there a case where the title won't go back to '/' ?

Reply:
its a good practice to have a model for navigation items... in case one day they change... we know what is changing and where